_quarto.yml

```
@@ -0,0 +1,125 @@
+project:
+  type: website
+  output-dir: docs
+  render:
+    - "documentation/index.md"
+    - "documentation/"
+    - "examples/"
+
+format:
+  html:
+    toc: true
+    theme:
+      light: [flatly, "quarto-materials/tiledb.scss"]
+    mainfont: Helvetica
+    fontsize: 1rem
+    linkcolor: "#4d9fff"
+    code-copy: true
+    code-overflow: wrap
```

Suggestion: Code wrapping makes several of the code blocks unreadable.

```suggestion
    code-overflow: scroll
```